README.md

```diff
@@ -57,19 +57,17 @@ A `Boundary` component receives props below:
 
 ```typescript
 interface BoundaryProps {
-    // What will this boundary component itself render, default to div
-    is: ReactElementType;
     // Defines how resource responses are cached in context, either per resource function or per invocation params
     cacheMode: 'function' | 'key';
     // When any of async progress is pending, boundary will render this element
     pendingFallback: Node;
     // When any error are received, will render this function
     renderError(error: Error): Node;
+    // When any error are catched, will call this function
+    componentDidCatch(error: Error, info: ErrorInfo, scope?: Scope): void;
```

Comment:
I'd like to name it `onErrorCaught` like any other callbacks

Reply:
Also I didn't think `scope` is necessary, errors are caught within a bounary, it should be reported no matter where (which scope) it is stored